### PR TITLE
Browser messages

### PR DESCRIPTION
**Pull request by wissog**

### CODE REVIEW

File: src/components/Sections/Home/LatestDailyLesson.js [hunk 1]
```diff
@@ -20,8 +20,9 @@ const getRandomImage = () => {
 };
 
 const LatestDailyLesson = ({ collection, t }) => {
-  // eslint-disable-next-line no-unused-vars
-  const [imageSrc, _] = useState(getRandomImage());
+  const [imageSrc, setImage] = useState();
```
Comment (gshilin) [inline src/components/Sections/Home/LatestDailyLesson.js:23]: useState requires parameter

File: src/components/Sections/Home/LatestDailyLesson.js [hunk 2]
```diff
@@ -20,8 +20,9 @@ const getRandomImage = () => {
...
+
+  useEffect(() => setImage(getRandomImage()), []);
```
Comment (gshilin) [inline src/components/Sections/Home/LatestDailyLesson.js:25]: I don't understand why this is better then initial value in useState()

File: src/components/Sections/Home/Container.js [hunk 3]
```diff
@@ -106,7 +105,6 @@ const HomePageContainer = ({ location, t }) => {
 
 HomePageContainer.propTypes = {
   location: shapes.HistoryLocation.isRequired,
-  f: PropTypes.func.isRequired,
 };
```
Comment (gshilin) [inline src/components/Sections/Home/Container.js:108]: Please think a bit... Look in browser...

File: src/components/Sections/Home/LatestDailyLesson.js (hunk 2 above)
Comment (wissog) [reply src/components/Sections/Home/LatestDailyLesson.js:25]: adding useEffect gets rid of a warning:
Warning: Prop `src` did not match. Server: "null" Client: "https://kabbalahmedia.info/imaginary/thumbnail?url=https%3A%2F%2Fkabbalahmedia.info%2Fassets%2Flessons%2Flatest_lesson_6.jpg&width=512&height=288&nocrop=false&stripmeta=true"
    in img (created by Image)
    in Image (at LatestDailyLesson.js:30)
    in a (created by LinkAnchor)
    in LinkAnchor (created by Context.Consumer)
    in Link (at MultiLanguageLinkCreator.js:32)
    in MultiLanguageLinkHOC (at LatestDailyLesson.js:29)
    in div (at LatestDailyLesson.js:28)
    in LatestDailyLesson (created by NamespacesConsumerComponent)
    in NamespacesConsumerComponent (created by WithMergedOptions)
    in WithMergedOptions (created by Context.Consumer)
    in WithMergedOptions(NamespacesConsumerComponent) (created by LoadNamespace)
    in LoadNamespace (created by WithMergedOptions)
    in WithMergedOptions (created by Context.Consumer)
    in LoadNamespace(LatestDailyLesson) (at HomePage.js:171)

File: src/components/Sections/Home/LatestDailyLesson.js (hunk 1 above)
Comment (wissog) [reply src/components/Sections/Home/LatestDailyLesson.js:23]: you may omit null

File: src/components/Sections/Home/Container.js (hunk 3 above)
Comment (wissog) [reply src/components/Sections/Home/Container.js:108]: missed t as well 🥇

File: src/components/Sections/Home/LatestDailyLesson.js (hunk 2 above)
Comment (gshilin) [reply src/components/Sections/Home/LatestDailyLesson.js:25]: Our target is not to delete messages, but understand what they are about, what clue they try to give us

File: src/components/Sections/Home/Container.js (hunk 3 above)
Comment (gshilin) [reply src/components/Sections/Home/Container.js:110]: It was a typo, please return this line and just replace f with t

**Comment by wissog** [comment]
yea i did a research about it and effect was found as a tool to get rid
from it


*Gabi*

*"What R we living for?" :)*

**Pull request merged**